OpenOversight/app/__init__.py

```diff
@@ -19,10 +19,12 @@
 from markupsafe import Markup
 
 from .config import config
+from .gmail_client import GmailClient
 
 
 bootstrap = Bootstrap()
 mail = Mail()
+GmailClient()
```

Comment:
This is added to the `__init__` file so that the build fails if the configuration file isn't there.